django/applications/catmaid/control/useranalytics.py

```diff
@@ -510,7 +510,7 @@ def eventsPerIntervalPerDayPlot(ax,times,start_date,end_date,interval=60):
 
     timeaxis = [i for i in range(24 * 60 / interval )]
     timelabels = []
-    for i in range(24 * 60 / 30):
+    for i in range(int(24 * 60 / 30)):
```

Comment:
In python3 I think this promotes to float for division, and range() can't operate on that.

Reply:
Good catch!